k8s/overlays/local/kustomization.yaml

```diff
@@ -114,3 +115,8 @@ configMapGenerator:
     behavior: merge
     files:
       - cassandra.properties
+
+  - name: directory-server-configmap
+    behavior: merge
+    files:
+      - cassandra.properties
```

Comment:
do you use cassandra too ?

Reply:
No, it's a mistake. Replaced by database.properties.